packages/opentelemetry-resources/src/detectors/BrowserDetector.ts

```diff
@@ -61,5 +62,18 @@ class BrowserDetector implements Detector {
   }
 }
 
+//Add Browser related attributes to resources
+export function getBrowserAttributes():ResourceAttributes {
```

Comment:
I don't think this function needs to be exported.

```suggestion
function getBrowserAttributes() : ResourceAttributes {
```